### ISSUE TITLE
transfer of maintainership

### ISSUE THREAD

**Issue by christian-rauch** [flathub/org.cloudcompare.CloudCompare#103]
@rjanvier I would like to pass on the maintainership of the `org.cloudcompare.CloudCompare` flatpak to you. I am not using CloudCompare that much anymore and you are the one most active in the flatpak repo anyway.

If you are fine with this, I would essentially unsubscribe from this repo and let you handle "everything". I believe you already have the same access rights as me, so you should be able to manage PRs and issues as I do.

Please let me know if this is ok with you.

**Comment by rjanvier** [comment]
Hi @christian-rauch, it's ok on my side if the current situation turned to be a burden for you, but feel free to continue if you want !
Thank you for kickstarting this package and for all the time and effort you’ve put in maintaining it over the years (especially during the two years I was away from CC).

**Comment by christian-rauch** [comment]
Yes, it kind of became a burden since the build process failed quite often from automatic dependency updates and I had to spend more time on this than I was initially willing to. Essentially, you are fixing these issues anyway and managing new versions.

@nedrichards @bilelmoussaoui Is there something that needs to be done to turn over the maintainership of a flatpak to someone else?

**Comment by tmontaigu** [comment]
Is this still relevant ? I am interested in publishing more recent versions of CC as well (based on the master branch)
